AngularJwtAuth/src/app/forms/conferenceForm/conferenceForm.component.ts

```diff
@@ -1,46 +1,64 @@
 import { Component, OnInit } from '@angular/core';
-import { FormBuilder, Validators, FormGroup, EmailValidator, FormArray } from '@angular/forms';
+import { FormBuilder, Validators, FormGroup, FormArray, AbstractControl } from '@angular/forms';
 import { forbiddenConferenceNameValidator } from './shared/conferenceName.validator';
-import { ConferenceDateValidator } from './shared/conferenceDate.validator';
 import { ConferenceService } from './conferenceForm.service';
 import { Conference } from './conferenceForm';
 import { MatDialogConfig, MatDialog, MatSnackBar } from '@angular/material';
 import { ConferenceFormDialogComponent } from './conferenceFormDialog/conferenceForm-dialog.component';
-import { Stage } from '../../objects/conference/stage';
+import { ConferenceDateValidator } from './shared/conferenceDate.validator';
 
 @Component({
   selector: 'conferenceForm',
   templateUrl: './conferenceForm.component.html',
   styleUrls: ['./conferenceForm.component.css']
 })
 export class ConferenceFormComponent implements OnInit {
-
   conferenceForm: FormGroup;
+  dateGroup: FormGroup;
   conferences: Conference[] = [];
   categories: string[] = [];
   stages: string[] = [];
+  minDate = new Date(Date.now());
```

Comment:
New date geeft volgens mij al de huidige datum dus is het niet nodig om de huidige datum opnieuw mee te geven.